src/views/AnalysisView.vue

```diff
@@ -87,7 +89,8 @@ const matrix = computed(() => {
     props.scale.head(MAX_SCALE_SIZE).mergeOptions({
       centsFractionDigits: 1,
       decimalFractionDigits: 3,
-    })
+    }), 
+    1 - (matrixStartIndex.value)
```

Comment:
You shouldn't call a property something and then pass it as the inverse of that to another function.

Reply:
What I have should work fine (for issue #411), though. I *am* using just "matrixStartIndex.value" in other parts.

Reply:
It's admittedly a really janky way to not display the zeroth column but just for the zero-indexed option. I should fix that.